chromium/product.yml

```diff
@@ -5,3 +5,9 @@ type: product
 benchmark_root: "./guide"
 
 profiles_root: "./profiles"
+
+cpes:
+  chromium:
+    name: "cpe:/a:google:chromium-browser"
+    title: "Google Chromium Browser"
+    check_id: installed_app_is_chromium
```

Comment:
Just like the `profiles_root`, I want to have `additional_cpes` in this file, because as a content author, I don't want to choose to add `cpes.yml` or not. I might come with my own `cpes.yml` file that doesn't contain what we have now.

Reply:
> So you'd like to be able to choose the file to source the CPE's from?
> 
> So if `product.yml` specified a directory, and any `yml` (or even `xml`) file there was sourced. Would it satisfy your needs?
> 
> ```
> cpes_root: "../shared/applicability"
> ```

It definitely would address the needs!

Reply:
Possibility to load multiple source CPE yml files added in 05506be